prov/verbs/src/verbs_init.c

```diff
@@ -784,12 +784,8 @@ VERBS_INI
 	ofi_hmem_init();
 	ofi_monitors_init();
 #endif
-
-	vrb_os_mem_support(&vrb_gl_data.peer_mem_support,
-			   &vrb_gl_data.dmabuf_support);
-
-	if (vrb_read_params() || vrb_os_ini() ||
-	    vrb_init_info(&vrb_util_prov.info))
+	if (vrb_os_ini())
```

Comment:
It looks odd to have os initialization in vrb_read_params.  Maybe combine vrb_os_ini() with vrb_os_mem_support()?

Reply:
The way github shows the changes is misleading due to the use of macro as the function name. This call is not inside vrb_read_params(), it's inside VERBS_INI.

Reply:
You're right.  It's more obvious when I look at the combined changes versus commit by commit.